apps/condo/domains/ticket/components/TicketForm/CreateTicketForm.tsx

```diff
@@ -32,7 +34,36 @@ export const CreateTicketActionBar = ({ handleSave, isLoading }) => {
                     const { property, details, placeClassifier, categoryClassifier, deadline } = getFieldsValue(REQUIRED_TICKET_FIELDS)
                     const disabledCondition = !property || !details || !placeClassifier || !categoryClassifier || !deadline
 
-                    return (
+                    return isSmall ? (
+                        <ActionBar isFormActionBar>
+                            <Col>
+                                <Space direction='vertical' size={24} >
```

Comment:
Only Row can me placed inside the Col element: https://ant.design/components/grid/